flexget/plugins/output/qbittorrent.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+
+from requests import Session
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('qbittorrent')
+
+
+class OutputQBitTorrent(object):
+    """
+    Example:
+
+      qbittorrent:
+        username: <USERNAME> (default: (none))
+        password: <PASSWORD> (default: (none))
+        host: <HOSTNAME> (default: localhost)
+        port: <PORT> (default: 8080)
+        movedone: <OUTPUT_DIR> (default: (none))
+        label: <LABEL> (default: (none))
+    """
+    schema = {
+        'anyOf': [
+            {'type': 'boolean'},
+            {
+                'type': 'object',
+                'properties': {
+                    'username': {'type': 'string'},
+                    'password': {'type': 'string'},
+                    'host': {'type': 'string'},
+                    'port': {'type': 'integer'},
+                    'movedone': {'type': 'string'},
```

Comment:
I'm thinking we should change this option to 'path' rather than movedone. Transmission works similarly, and that's the option name in that plugin.